DDA_Line_Drawing_Algorithm/DDA_Line_Drawing_Algo.c

```diff
@@ -0,0 +1,40 @@
+#include<stdio.h>
+#include<conio.h>
+#include<stdlib.h>
+#include<graphics.h>
+void main()
+{
+	float x1,x2,y1,y2,dx,dy,steps,xi,yi,i;
```

Comment:
Please use 4 space indentation in all files.

Reply:
added 4 space indentation 
now have a look once

Reply:
Not fixed.

Reply:
check now